app/src/test/java/com/nextcloud/talk/utils/UserIdUtilsTest.kt

```diff
@@ -0,0 +1,36 @@
+package com.nextcloud.talk.utils
```

Comment:
Please add the usual license header as a comment, see for example https://github.com/nextcloud/talk-android/blob/master/app/src/main/java/com/nextcloud/talk/activities/BaseActivity.kt#L1-L21